### DISCUSSION TITLE
stamp/watermark command description argument

### DISCUSSION THREAD

**Discussion by kharabela** [pdfcpu/pdfcpu#691] [Q&A]
I am using this to print watermark and it does appear on the top left.
What all combinations of these parameters can I use?Is there any reference to go through?
desc := "pos:tl, points:22, rot:0, scale:1 abs, off:0 -5, opacity:0.5"
	wm, err := pdfcpu.TextWatermark("This is a watermark", desc, onTop, false, types.POINTS)

**Comment by hhrutter** [comment]
https://pdfcpu.io/core/watermark
